backend/src/components/fundingAgreements.js

```diff
@@ -32,7 +33,8 @@ async function getFundingAgreements(req, res) {
     })
 
     if (req.query?.includeTopUp) {
-      const topUps = await getTopUpFundingByFilter(filter)
+      const topUpFilter = filter + ` and statuscode ne ${TOP_UP_FUNDING_STATUS_CODES.DRAFT}`
```

Comment:
It's a little odd to combine string concatenation and template literals. You can just include filter in the template literal.